vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentalGroupsLayout.java

```diff
@@ -98,14 +121,12 @@ public ExperimentalGroupCard(int sampleSize, VariableLevel... variableLevels) {
       tagsContainer.setFlexWrap(FlexWrap.WRAP);
       tagsContainer.setFlexDirection(FlexDirection.ROW);
       tagsContainer.setAlignContent(ContentAlignment.START);
+      tagsContainer.addClassNames(Overflow.HIDDEN);
+      tagsContainer.setWidth("165px");//TODO better way to hide overflow?
```

Comment:
This should be set to use the full width of the card container 
```suggestion
      tagsContainer.setWidthFull()
```